src/components/Header/SearchBar.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const SearchBar = () => {
+  return (
+    <div className="flex-1 items-center justify-center">
+      <input
```

Comment:
white background shadow yen beda 

ee tara ne erli
![image](https://github.com/user-attachments/assets/e36b80b5-f4b0-4d6b-8c26-5c5a1c422ae8)
width bekadre reduce madi nodu chanag kansdre 

amele search icon miss ide nodu

Reply:
and text-body ( font-size 14px )
yella kade consistent ag erutte aga